DEVELOPMENT.md

```diff
@@ -70,3 +70,39 @@ or directly.
 A `.clang-format` file which defines our specific code style lives in the
 top level directory of the repository. The style is identical to the Google
 style.
+
+## Cross-Compiling for GGP
+
+_Note:_ This was only tested on Linux. But cross compilation on Windows
+should work as well.
+
+_Note:_ Cross compiling the UI is not supported.
+
+First we need to cross compile all the dependencies:
```

Comment:
So are these all manual steps? Maybe we should add a script for this (at least for the build part)?

Reply:
That's what I was referring to in my initial comment. I wasn't sure if I should add a script. What is your opinion on that?

Reply:
If it's possible to have a script that works reliably I'm in favor of it.

Reply:
I added a script (`bootstrap-orbit-ggp.sh`). 

I will resolve this comment for now. If you need anything else, feel free to create further comments!